docs/servo_power_monitor/src/index.ts

```diff
@@ -7,6 +7,7 @@ import Dygraph from 'dygraphs';
 import moment from 'moment';
 
 const intervalMs = 100;
+const cancel_cmd = '\x03\n';
```

Comment:
Please use CONSTANT_CASE instead of snake_case for constants
https://google.github.io/styleguide/tsguide.html#naming-rules-by-identifier-type

Reply:
should I also fix `intervalMs` to `INTERVAL_MS`?

Reply:
ideally, it should be separated (and the css changes as well), but as far as it is simple enough, okay to change them altogether!

Reply:
Got it, fixed!